account_reconciliation_widget/models/account_bank_statement.py

```diff
@@ -256,6 +257,9 @@ def _create_counterpart_and_new_aml(
             self.move_id.with_context(
                 skip_account_move_synchronization=True
             ).action_post()
+        elif wo_aml.filtered(lambda l: l.analytic_account_id):
```

Comment:
Please take into account analytic_tag_ids too, same problem

Reply:
@jonerikceberio Could you make that change @omar7r is asking?